kythe/go/extractors/config/runextractor/mavencmd/mavencmd.go

```diff
@@ -82,10 +84,27 @@ func (m *mavenCommand) Execute(ctx context.Context, fs *flag.FlagSet, args ...in
 	if err := modifier.PreProcessPomXML(m.pomXML); err != nil {
 		return m.Fail("error modifying maven pom XML %s: %v", m.pomXML, err)
 	}
-	if err := exec.Command("mvn", "clean", "install",
+
+	// Print diff to show changes made to pom.xml.
+	log.Printf("Modified pom.xml. Diff:")
```

Comment:
Can this be put behind a -verbose flag or something?

Reply:
Done.